src/main/java/org/folio/service/export/storage/AWSStorageServiceImpl.java

```diff
@@ -122,4 +130,30 @@ public void storeFile(FileDefinition fileDefinition, String tenantId) {
     }
   }
 
+  @Override
+  public void removeFilesRelatedToJobExecution(JobExecution jobExecution, String tenantId) {
+    String bucketName = getProperty(BUCKET_PROP_KEY);
+    if (StringUtils.isNullOrEmpty(bucketName)) {
+      throw new ServiceException(HttpStatus.HTTP_FORBIDDEN, ErrorCode.S3_BUCKET_NAME_NOT_FOUND.getDescription());
```

Comment:
Maybe it will be better to use `HttpStatus.HTTP_NOT_FOUND` instead of `HttpStatus.HTTP_FORBIDDEN` ?

Reply:
agree, updated, thanks)